### PR TITLE
Add moreutils recipe

### PR DESCRIPTION
**Pull request by shahin**

### CODE REVIEW

File: recipes/moreutils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  url: http://mirrors.kernel.org/debian/pool/main/m/{{ pkgname }}/{{ pkgname }}_{{ version }}.orig.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+test:
+  commands:
+    - sponge -h
```
Comment (patricksnape) [inline recipes/moreutils/meta.yaml]: Is this the only command?

File: recipes/moreutils/build.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+make PREFIX=$PREFIX MANS=;
+make install PREFIX=$PREFIX MANS=README
```
Comment (ocefpaf) [inline recipes/moreutils/build.sh]: Is there a `make check` that we can run here?

File: recipes/moreutils/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+about:
+  home: http://joeyh.name/code/moreutils/
+  license: GPL-2.0
+  summary: a growing collection of the unix tools that nobody thought to write long ago when unix was young
```
Comment (ocefpaf) [inline recipes/moreutils/meta.yaml]: To help out bots create a nice `README` on the feedstock can you start with an upper case letter (`A`) and end with a full stop (`.`)?

File: recipes/moreutils/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/moreutils/meta.yaml]: Would figure `moreutils` would have more? Looks like there is quite a [list]( https://joeyh.name/code/moreutils/ ). Let's add some test commands for them too.

**Comment by jakirkham** [comment]
Any thoughts on the comments above, @shahin?

**Comment by jakirkham** [comment]
Friendly nudge @shahin .

**Pull request closed without merging**